src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/order/Price.java

```diff
@@ -0,0 +1,78 @@
+package ay2122s1_cs2103t_w16_2.btbb.model.order;
+
+import static ay2122s1_cs2103t_w16_2.btbb.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the price of a recipe in BTBB.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Price {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Price should only contain numbers, it should be positive "
+                    + "and the largest acceptable price is " + Float.MAX_VALUE + ".";
+    private final float price;
+
+    /**
+     * Constructs a {@code Price}.
+     *
+     * @param price A valid price.
+     */
+    public Price(String price) {
+        requireNonNull(price);
+        checkArgument(isValidPrice(price), MESSAGE_CONSTRAINTS);
+        this.price = Float.parseFloat(price);
```

Comment:
Can we use regex for this. Otherwise we can have duplicate price inputs 2.113 and 2.114. These get parsed to 2.11 in storage. And when the app is reloaded it crashes.